src/components/ValueAnnotation.tsx

```diff
@@ -1,7 +1,54 @@
+import { useState } from 'react';
+import { Typography, Paper as Card } from '@mui/material';
+import useDataStore from '../stores/data';
+import SideColumnNavBar from './SideColumnNavBar';
+import Categorical from './Categorical';
+import Continuous from './Continuous';
+
 function ValueAnnotation() {
+  const columns = useDataStore((state) => state.columns);
+  const [selectedColumnId, setSelectedColumnId] = useState<string | null>(null);
+  const [selectedColumnDataType, setSelectedColumnDataType] = useState<
+    'Categorical' | 'Continuous' | null
+  >(null);
```

Comment:
Why is that state? Can't we derive the columnDataType from `columns` (from store) + `columnID`?